templates/Pagination/twitter_bootstrap_v4_material_design_icons_sortable_link.html.twig

```diff
@@ -0,0 +1,25 @@
+{#
+/**
+ * @file
+ * Twitter Bootstrap - Material Design Icons Sorting control implementation.
```

Comment:
This is not related with Bootstrap, right?

Reply:
Yes/no: Bootstrap is due to the CSS class "float-right" in line 13